server/notification-providers/gtx-messaging.js

```diff
@@ -0,0 +1,38 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class GtxMessaging extends NotificationProvider {
+    name = "gtxmessaging";
+
+    /**
+     * @inheritDoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+
+        let authKey = notification.gtxMessagingApiKey;
+        let from = notification.gtxMessagingFrom.trim();
+        let to = notification.gtxMessagingTo.trim();
+        let text = msg.replaceAll("🔴 ", "").replaceAll("✅ ", "");
+
+        try {
+
+            let data = new URLSearchParams();
```

Comment:
If a variable name is not mutated, please prefer `const`

```suggestion
            const data = new URLSearchParams();
```